test/cookbook-test.js

```diff
@@ -5,23 +5,113 @@ import recipeData from '../src/data/recipes';
+            "quantity": {
+              "amount": 1.5,
+              "unit": "c"
+            }
+          },
+          {
+            "name": "baking soda",
+            "id": 18372,
+            "quantity": {
+              "amount": 0.5,
+              "unit": "tsp"
+            }
+          }
+        ],
+        "instructions": [
+          {
+            "number": 1,
+            "instruction": "In a large mixing bowl, whisk together the dry ingredients (flour, pudding mix, soda and salt). Set aside.In a large mixing bowl of a stand mixer, cream butter for 30 seconds. Gradually add granulated sugar and brown sugar and cream until light and fluffy."
+          },
+          {
+            "number": 2,
+            "instruction": "Add egg and vanilla and mix until combined."
+          }
+        ],
+        "tags": [
+          "antipasti",
+          "starter",
+          "snack",
+          "appetizer",
+          "antipasto",
+          "hor d'oeuvre"
+        ]
+      };
+      dummyRecipe2 =  {
+          "name": "stuff cake",
+          "id": 595737,
+          "image": "https://spoonacular.com/recipeImages/595736-556x370.jpg",
+          "ingredients": [
+            {
+              "name": "cake flour",
+              "id": 20082,
+              "quantity": {
+                "amount": 1.5,
+                "unit": "c"
+              }
+            },
+            {
+              "name": "baking soda",
+              "id": 18372,
+              "quantity": {
+                "amount": 0.5,
+                "unit": "tsp"
+              }
+            }
+          ],
+          "instructions": [
+            {
+              "number": 1,
+              "instruction": "In a large mixing bowl, whisk together the dry ingredients (flour, pudding mix, soda and salt). Set aside.In a large mixing bowl of a stand mixer, cream butter for 30 seconds. Gradually add granulated sugar and brown sugar and cream until light and fluffy."
+            },
+            {
+              "number": 2,
+              "instruction": "Add egg and vanilla and mix until combined."
+            }
+          ],
+          "tags": [
+            "antipasti",
+            "starter",
+            "snack",
+            "appetizer",
+            "antipasto",
+            "hor d'oeuvre",
+            "test"
+          ]
+        };
+      cookbook = new Cookbook([dummyRecipe1, dummyRecipe2]);
   });
 
+
   it('Should have an array of all recipes', () => {
-    expect(cookbook.recipes).to.be.an('array');
+    expect(cookbook.recipes).to.deep.equal([dummyRecipe1, dummyRecipe2]);
   });
 
-  describe('findRecipe', () => {
-    it('Should be able to filter through its array by ingredients', () => {
-      expect(cookbook.findRecipe('yolk').length).to.equal(2);
-    });
+  it('Should be able to filter through ingredients and return an array of recipes', () => {
+    expect(cookbook.findRecipe('baking soda')).to.deep.equal([dummyRecipe1, dummyRecipe2]);
+  });
```

Comment:
Again, glad to see a more flexible test looking for the actual contents returned and not just a similar length array.